### ISSUE TITLE
Support `legibilityWeight` accessibility option

### ISSUE THREAD

**Issue by PavelHolec** [kiwicom/orbit-swiftui#503]
`Text` should support the `legibilityWeight` option. The default weight should be replaced with `bold` when enabled.
